data_definition_postgres.sql

```diff
@@ -164,8 +170,8 @@ create table ethical_problem (
 );
 
 insert into ethical_problem (title) values
-    ('deforestation'),
-    ('carbon emission');
+    ('Deforestation -  the decrease in forest areas across the world that are lost for other uses such as agricultural croplands, urbanization, or mining activities.'),
+    ('Carbon Emission - the release of carbon into the atmosphere; the main contributors to climate change.');
```

Comment:
We should not include the description of an ethical problem in the title field. Why don't we create a separate description field? Another option is to include the description in the `ingredient_ethical_problem` table. That way, we can more specifically describe how a particular ingredient causes that specific ethical problem.

Reply:
This is updated. I created a field for the description.